cmd/wait-for-github/ci.go

```diff
@@ -0,0 +1,123 @@
+// wait-for-github
```

Comment:
Licenses annoy me personally. I don't think we need those if you have the license declared top-level. You only need to declare if the license of a file differs from the whole repo

Reply:
I did fix all of your other comments, thanks for those, but I'm going to leave this one. I like the practice. It means that someone can take any of the source files out of the project and use it for their own stuff if they want and the license information travels with it automatically.